http/oidc/src/test/java/org/wildfly/security/http/oidc/OidcTest.java

```diff
@@ -909,5 +940,22 @@ static InputStream getTenantConfigWithProviderUrl(String tenant) {
     private static final String getClientPageTestForTenant(String tenant) {
         return tenant.equals(TENANT1_ENDPOINT) ? TENANT1_ENDPOINT : TENANT2_ENDPOINT + ":" + CLIENT_PAGE_TEXT;
     }
+
+    private InputStream getOidcRefreshConfigurationInputStream() {
+        return getOidcRefreshConfigurationInputStream(CLIENT_SECRET, KEYCLOAK_CONTAINER.getAuthServerUrl());
+    }
+    private InputStream getOidcRefreshConfigurationInputStream(String clientSecret, String authServerUrl) {
+        String oidcConfig = "{\n" +
+                "    \"" + REALM + "\" : \"" + TEST_REFRESH_REALM + "\",\n" +
+                "    \"" + RESOURCE + "\" : \"" + CLIENT_ID + "\",\n" +
+                "    \"" + PUBLIC_CLIENT + "\" : \"false\",\n" +
+                "    \"" + AUTH_SERVER_URL + "\" : \"" + authServerUrl + "\",\n" +
```

Comment:
It would be good to use the client-id, provider-url configuration instead in this config.

Reply:
Fixed